lib/ansible/modules/system/vdo.py

```diff
@@ -606,14 +606,18 @@ def run_module():
         modtrans = {}
 
         for statfield in statusparamkeys:
-            currentvdoparams[statfield] = processedvdos[desiredvdo][statfield]
-            modtrans[statfield] = vdokeytrans[statfield]
+            if statfield in processedvdos[desiredvdo]:
+                currentvdoparams[statfield] = processedvdos[desiredvdo][statfield]
+                modtrans[statfield] = vdokeytrans[statfield]
+
+            if statfield not in processedvdos[desiredvdo]:
```

Comment:
why not just `else:`?

Reply:
No reason; I just forgot.  We can use `else:`.

Reply:
Should be all set now; I just tested it on my machine to confirm.